docs/GettingStarted/configuration/PelorusExporters.md

```diff
@@ -80,205 +80,93 @@ This is the list of options that can be applied to `exporters.instances` section
+: **Recommended for sensitive data**
+
+: List of secrets, like in the following example.
+```yaml
+env_from_secrets:
+- example-secret
+- other-secret
+```
+[More examples](#secrets).
+
+: Check the list of all available options per exporter type:
+: - [Deploy time](ExporterDeploytime.md#deploy-time-exporter-configuration-options)
+- [Commit time](ExporterCommittime.md#commit-time-exporter-configuration-options)
+- [Failure](ExporterFailure.md#failure-time-exporter-configuration-options)
+
 
 ###### env_from_configmaps
 
 - **Required:** no
 - **Type:** list
 
-    List of ConfigMaps, like in the following example.
-    ```yaml
-    env_from_configmaps:
-    - example-configmap
-    - other-configmap
-    ```
-
-    Check the list of all available options per exporter type:
-
-    - [Deploy time](ExporterDeploytime.md#deploy-time-exporter-configuration-options)
-    - [Commit time](ExporterCommittime.md#commit-time-exporter-configuration-options)
-    - [Failure](ExporterFailure.md#failure-time-exporter-configuration-options)
-
-    **ConfigMap Examples**
-
-    1. To create a ConfigMap named **example-config** in **pelorus** namespace, with the
-
-        - option **APP_LABEL** with value **example**
-        - option **NAMESPACES** with value **one,two**
-
-        using the file **config.yaml**
-        ```yaml
-        apiVersion: v1
-        kind: ConfigMap
-        metadata:
-          name: example-config
-          namespace: pelorus
-        data:
-          APP_LABEL: "example"
-          NAMESPACES: "one,two"
-        ```
-        run
-        ```
-        oc apply -f config.yaml
-        ```
-        Then, add
-        ```yaml
-        [...]
-                env_from_configmaps:
-                - example-config
-        [...]
-        ```
-        to the exporter configuration.
-
-        If no `metadata.namespace` is added to ConfigMap file, you must run the command with the namespace you want to apply it. For example, `oc apply -f config.yaml -n pelorus`.
-
-        More [examples ConfigMap files](https://github.com/konveyor/pelorus/tree/master/charts/configmaps).
-
-    1. To create a ConfigMap named **example-for-all** in **pelorus** namespace, that will be used by multiple exporters, with the option **LOG_LEVEL** with value **DEBUG**, using the file **config.yaml**
-        ```yaml
-        apiVersion: v1
-        kind: ConfigMap
-        metadata:
-          name: example-for-all
-          namespace: pelorus
-        data:
-          LOG_LEVEL: "DEBUG"
-        ```
-        run
-        ```
-        oc apply -f config.yaml
-        ```
-        Then, add
-        ```yaml
-        [...]
-                env_from_configmaps:
-                - example-for-all
-        [...]
-        ```
-        to each one of the exporters configuration.
+: List of ConfigMaps, like in the following example.
+```yaml
+env_from_configmaps:
+- example-configmap
+- other-configmap
+```
+[More examples](#configmaps).
```

Comment:
Same here. I had to go to the PR source to see how it's being referenced from this section, because I missed it.